### ISSUE TITLE
Crash in PushRegistratorADM.fireCallback

### ISSUE THREAD

**Issue by rampr** [OneSignal/OneSignal-Android-SDK#92]
Hi,

We are seeing a lot of these errors in our Amazon app. Error message below. I can try to get more information if it is needed.

```
java.lang.NullPointerException
    at com.onesignal.PushRegistratorADM.fireCallback(PushRegistratorADM.java:67)
    at com.onesignal.ADMMessageHandler.onRegistered(ADMMessageHandler.java:55)
    at com.amazon.device.messaging.ADMMessageHandlerBase.handleNewRegistrationID(ADMMessageHandlerBase.java:116)
    at com.amazon.device.messaging.ADMMessageHandlerBase.handleRegistrationChange(ADMMessageHandlerBase.java:106)
    at com.amazon.device.messaging.ADMMessageHandlerBase.onHandleIntent(ADMMessageHandlerBase.java:81)
    at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:135)
    at android.os.HandlerThread.run(HandlerThread.java:61)
```

**Comment by jkasten2** [comment]
@rampr Thanks for the report, added null checking for this in commit be4baaa4b23012ea858fa556b849a20aefe0caaf.

**Comment by rampr** [comment]
Thanks @jkasten2, When can we expect a release with this fix.

**Comment by jkasten2** [comment]
@rampr This fix is include in [version 3.2.0](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/3.2.0).
